index.android.js

```diff
@@ -82,16 +82,18 @@ export default class FlowApp extends Component {
             </NavDrawerAndroid>
           );
         }}
+
       />
     );
   }
 
-
+//this doesn't work, it can't get the Navigator from the above code,
```

Comment:
I fixed this in my branch :) I found a slightly better way to deal with logouts that what I was trying before that ended up working well.